chain/rosetta-rpc/src/adapters/nep141.rs

```diff
@@ -0,0 +1,209 @@
+                        let base = get_base(Event::Nep141, outcome, block_header)?;
+                        let custom = crate::models::FtEvent {
+                            affected_id: AccountIdentifier::from_str(&transfer_event.old_owner_id)?,
+                            involved_id: Some(AccountIdentifier::from_str(
+                                &transfer_event.new_owner_id,
+                            )?),
+                            delta: -transfer_event.amount.parse::<i64>()?,
+                            cause: "TRANSFER".to_string(),
+                            memo: transfer_event
+                                .memo
+                                .as_ref()
+                                .map(|s| s.escape_default().to_string()),
+                            symbol: currency.symbol.clone(),
+                            decimals: currency.decimals,
+                        };
+                        ft_events.push(build_event(base, custom).await?);
+
+                        let base = get_base(Event::Nep141, outcome, block_header)?;
+                        let custom = crate::models::FtEvent {
+                            affected_id: AccountIdentifier::from_str(&transfer_event.new_owner_id)?,
+                            involved_id: Some(AccountIdentifier::from_str(
+                                &transfer_event.old_owner_id,
+                            )?),
+                            delta: transfer_event.amount.parse::<i64>()?,
+                            cause: "TRANSFER".to_string(),
+                            memo: transfer_event
+                                .memo
+                                .as_ref()
+                                .map(|s| s.escape_default().to_string()),
+                            symbol: currency.symbol.clone(),
+                            decimals: currency.decimals,
+                        };
+                        ft_events.push(build_event(base, custom).await?);
+                    }
+                }
+            }
+        }
+    }
+    Ok(ft_events)
+}
+
+pub(crate) async fn get_fungible_token_balance_for_account(
+    view_client_addr: &actix::Addr<near_client::ViewClientActor>,
+    block_header: &near_primitives::views::BlockHeaderView,
+    contract_address: &String,
+    account_identifier: &AccountIdentifier,
+) -> crate::errors::Result<u128> {
+    let method_name = "ft_balance_of".to_string();
+    let account_id_for_args = account_identifier.clone().address.to_string();
+    let args = serde_json::json!({
+        "account_id":account_id_for_args,
+    })
+    .to_string()
+    .into_bytes();
+    let block_reference =
+        near_primitives::types::BlockReference::BlockId(BlockId::Hash(block_header.hash));
+    let request = near_primitives::views::QueryRequest::CallFunction {
+        account_id: near_account_id::AccountId::from_str(contract_address)?,
+        method_name,
+        args: near_primitives::types::FunctionArgs::from(args),
```

Comment:
similarly here, nothing wrong with `from` but `args.into()` could infer the type for you